modules/noti/slack_client.py

```
@@ -0,0 +1,110 @@
+def _cooldown_active(
+    redis_client: object, cooldown_key: str | None, cooldown_sec: int
+) -> bool:
+    """cooldown_key 가 주어지고 Redis 에 이미 존재하면 True 반환. 없으면 SET 후 False."""
+    if not cooldown_key or redis_client is None:
+        return False
+    key = f"{_COOLDOWN_PREFIX}{cooldown_key}"
+    try:
+        # SET NX EX: 없을 때만 설정 + TTL. 이미 있으면 False.
+        acquired = redis_client.client.set(key, "1", nx=True, ex=cooldown_sec)  # type: ignore[attr-defined]
+        return bool(acquired is None or acquired is False)
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)